src/Algolia.Search.Test/EndToEnd/Index/QueryRulesTest.cs

```diff
@@ -150,8 +150,27 @@ public async Task RulesTest()
                 Consequence = new Consequence { Params = new ConsequenceParams { Filters = "model:One" } }
             };
 
+            Rule ruleToSave5 = new Rule
```

Comment:
A more CSharpy syntax:

```suggestion
            var ruleToSave5 = new Rule
```

Reply:
Thanks! I'll open another PR to address that: there are more instance to fix in this file 😬